### PR TITLE
Add migration section into ingress-gateway using sds approach

### PR DESCRIPTION
**Pull request by JimmyCYJ**
Add short description about migration from file mount based approach to SDS approach.

### CODE REVIEW

**Comment by JimmyCYJ** [comment]
cc @myidpt

File: content/docs/tasks/traffic-management/secure-ingress/sds/index.md [hunk 1]
```diff
@@ -31,6 +31,12 @@ and the environment variables `INGRESS_HOST` and `SECURE_INGRESS_PORT` set.
     should work correctly with the instructions in this task. Otherwise, try
     a different implementation of `curl`, for example on a Linux machine.
 
+## Migration from file mount-based approach
```
Comment (myidpt) [inline content/docs/tasks/traffic-management/secure-ingress/sds/index.md]:
```suggestion
## (Optional) Migration from file mount-based approach
```

File: content/docs/tasks/traffic-management/secure-ingress/sds/index.md [hunk 2]
```diff
@@ -31,6 +31,12 @@ and the environment variables `INGRESS_HOST` and `SECURE_INGRESS_PORT` set.
...
+
+If you have configured an ingress gateway using the [file mount-based approach](/docs/tasks/traffic-management/secure-ingress/mount),
+and you want to migrate your ingress gateway to use SDS approach. There are no
```
Comment (myidpt) [inline content/docs/tasks/traffic-management/secure-ingress/sds/index.md]:
```suggestion
and you want to migrate your ingress gateway to use the SDS approach. There are no
```

File: content/docs/tasks/traffic-management/secure-ingress/sds/index.md [hunk 3]
```diff
@@ -31,6 +31,12 @@ and the environment variables `INGRESS_HOST` and `SECURE_INGRESS_PORT` set.
     should work correctly with the instructions in this task. Otherwise, try
     a different implementation of `curl`, for example on a Linux machine.
 
+## (Optional) Migration from file mount-based approach
```
Comment (rcaballeromx) [inline content/docs/tasks/traffic-management/secure-ingress/sds/index.md]:
```suggestion
{{< tip >}}
```

File: content/docs/tasks/traffic-management/secure-ingress/sds/index.md [hunk 4]
```diff
@@ -31,6 +31,12 @@ and the environment variables `INGRESS_HOST` and `SECURE_INGRESS_PORT` set.
...
+
+If you have configured an ingress gateway using the [file mount-based approach](/docs/tasks/traffic-management/secure-ingress/mount),
```
Comment (rcaballeromx) [inline content/docs/tasks/traffic-management/secure-ingress/sds/index.md]:
```suggestion
If you configured an ingress gateway using the [file mount-based approach](/docs/tasks/traffic-management/secure-ingress/mount),
```

File: content/docs/tasks/traffic-management/secure-ingress/sds/index.md [hunk 5]
```diff
@@ -31,6 +31,12 @@ and the environment variables `INGRESS_HOST` and `SECURE_INGRESS_PORT` set.
...
+and you want to migrate your ingress gateway to use the SDS approach. There are no
+extra steps required for the migration. You can start from the steps below.
```
Comment (rcaballeromx) [inline content/docs/tasks/traffic-management/secure-ingress/sds/index.md]:
```suggestion
extra steps required.
{{< /tip >}}
```

**Pull request merged**